apparmor.d/profiles-s-z/unbound

```diff
@@ -0,0 +1,64 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2021-2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Barmogund <set508@proton.me>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/unbound
+profile unbound @{exec_path} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/bus-system>
+  include <abstractions/bus/org.freedesktop.login1>
+  include <abstractions/nameservice-strict>
+  include <abstractions/p11-kit>
+  include <abstractions/ssl_certs>
+  include <abstractions/common/systemd>
+
+  capability net_bind_service,
+  capability net_raw,
+  capability sys_resource,
+  capability sys_chroot,
+  capability setgid,
+  capability setuid,
+  capability setpcap,
```

Comment:
According to https://github.com/NLnetLabs/unbound/blob/733d5f7161c5675e66e39974b912a08eeb48140c/contrib/unbound.service.in#L56

setpcap is not needed.